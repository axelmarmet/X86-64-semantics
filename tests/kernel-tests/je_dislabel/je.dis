0x8049000:	0x39c3                                      	cmpl   %eax,%ebx
0x8049002:	0x7406                                      	je     0x804900a <other_label>
0x8049004:	0x0b0501000000                              	orl    $0x1,%eax
0x804900a:	0x0b0502000000                              	orl    $0x2,%eax