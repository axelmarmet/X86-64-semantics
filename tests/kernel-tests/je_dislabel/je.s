 .text
.globl _start

_start:
    cmp %eax, %ebx
    je other_label
    or 1, %eax
other_label:
    or 2, %eax 
