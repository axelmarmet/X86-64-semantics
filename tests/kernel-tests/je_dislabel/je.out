<generatedTop>
  <k>
    fetch
  </k>
  <entrypoint>
    64'4198400
  </entrypoint>
  <exitpoint>
    64'4198408
  </exitpoint>
  <functargets>
    .Map
  </functargets>
  <rotargets>
    .Map
  </rotargets>
  <datatargets>
    .Map
  </datatargets>
  <bsstargets>
    .Map
  </bsstargets>
  <nextLocPc>
    64'0
  </nextLocPc>
  <lastseenlabel>
    .
  </lastseenlabel>
  <currentsection>
    "text"
  </currentsection>
  <sizeHint>
    hint ( 64'134516736 |-> 16
    64'134516738 |-> 16
    64'134516740 |-> 48
    64'134516746 |-> 48 )
  </sizeHint>
  <memstate>
    <text>
      code ( 16'14787 |-> execinstr ( cmpl %eax , %ebx , .Operands )
      16'29702 |-> execinstr ( je 0x804900a < other_label . .DisLabel > , .Operands )
      48'12116119519232 |-> execinstr ( orl $ 0x1 , %eax , .Operands )
      48'12116136296448 |-> execinstr ( orl $ 0x2 , %eax , .Operands ) )
    </text>
    <stack>
      <memory>
        <nextLoc>
          1
        </nextLoc>
        <objects>
          mem ( loc ( 134516736 , 1 , 8 , 0 ) |-> byte ( 0 , 16'14787 )
          loc ( 134516737 , 1 , 8 , 0 ) |-> byte ( 1 , 16'14787 )
          loc ( 134516738 , 1 , 8 , 0 ) |-> byte ( 0 , 16'29702 )
          loc ( 134516739 , 1 , 8 , 0 ) |-> byte ( 1 , 16'29702 )
          loc ( 134516740 , 1 , 8 , 0 ) |-> byte ( 0 , 48'12116119519232 )
          loc ( 134516741 , 1 , 8 , 0 ) |-> byte ( 1 , 48'12116119519232 )
          loc ( 134516742 , 1 , 8 , 0 ) |-> byte ( 2 , 48'12116119519232 )
          loc ( 134516743 , 1 , 8 , 0 ) |-> byte ( 3 , 48'12116119519232 )
          loc ( 134516744 , 1 , 8 , 0 ) |-> byte ( 4 , 48'12116119519232 )
          loc ( 134516745 , 1 , 8 , 0 ) |-> byte ( 5 , 48'12116119519232 )
          loc ( 134516746 , 1 , 8 , 0 ) |-> byte ( 0 , 48'12116136296448 )
          loc ( 134516747 , 1 , 8 , 0 ) |-> byte ( 1 , 48'12116136296448 )
          loc ( 134516748 , 1 , 8 , 0 ) |-> byte ( 2 , 48'12116136296448 )
          loc ( 134516749 , 1 , 8 , 0 ) |-> byte ( 3 , 48'12116136296448 )
          loc ( 134516750 , 1 , 8 , 0 ) |-> byte ( 4 , 48'12116136296448 )
          loc ( 134516751 , 1 , 8 , 0 ) |-> byte ( 5 , 48'12116136296448 ) )
        </objects>
        <freed>
          .Set
        </freed>
      </memory>
      <stackbase>
        stackBaseInfo ( . , . )
      </stackbase>
      <robase>
        roBaseInfo ( . , . , . )
      </robase>
      <database>
        dataBaseInfo ( . , . , . )
      </database>
      <bssbase>
        bssBaseInfo ( . , . , . )
      </bssbase>
    </stack>
  </memstate>
  <environment>
    <argc>
      .
    </argc>
    <argv>
      .
    </argv>
  </environment>
  <processormode>
    b32
  </processormode>
  <cpuidinfo>
    <processorsignature>
      "extendedfamily" |-> 8'0
      "extendedmodel" |-> 4'8
      "family" |-> 4'6
      "model" |-> 4'14
      "processortype" |-> 2'0
      "stepping" |-> 4'10
    </processorsignature>
    <extendedfunctioninfo>
      "1GBAv" |-> 1'1
      "64MODEAv" |-> 1'1
      "EDB" |-> 1'1
      "LAHF64" |-> 1'1
      "LZCNT" |-> 1'1
      "PREFETCHW" |-> 1'1
      "RDTSCP" |-> 1'1
      "SYSCALL" |-> 1'1
    </extendedfunctioninfo>
  </cpuidinfo>
  <cpuidavailable>
    true
  </cpuidavailable>
  <regstate>
    "AF" |-> 1'0
    "AM" |-> 1'0
    "CD" |-> 1'0
    "CF" |-> 1'0
    "CR2" |-> 32'0
    "CS" |-> 16'0
    "DE" |-> 1'0
    "DF" |-> 1'0
    "DS" |-> 16'0
    "EFLAGS" |-> 32'0
    "EM" |-> 1'0
    "ES" |-> 16'0
    "ET" |-> 1'1
    "FS" |-> 16'0
    "FSGBASE" |-> 1'0
    "GDTR_BASE" |-> 64'0
    "GDTR_LIMIT" |-> 16'0
    "GS" |-> 16'0
    "IA32_EFER" |-> 64'0
    "MCE" |-> 1'0
    "MP" |-> 1'0
    "NE" |-> 1'0
    "NW" |-> 1'0
    "OF" |-> 1'0
    "OSFXSR" |-> 1'0
    "OSXMMEXCPT" |-> 1'0
    "OSXSAVE" |-> 1'0
    "PAE" |-> 1'0
    "PCD" |-> 1'0
    "PCE" |-> 1'0
    "PCIDE" |-> 1'0
    "PE" |-> 1'1
    "PF" |-> 1'1
    "PG" |-> 1'0
    "PGE" |-> 1'0
    "PKE" |-> 1'0
    "PSE" |-> 1'0
    "PVI" |-> 1'0
    "PWT" |-> 1'0
    "PageDirectoryBase" |-> 20'0
    "R8" |-> 64'0
    "R9" |-> 64'0
    "R10" |-> 64'0
    "R11" |-> 64'0
    "R12" |-> 64'0
    "R13" |-> 64'0
    "R14" |-> 64'0
    "R15" |-> 64'0
    "RAX" |-> 64'0
    "RBP" |-> 64'0
    "RBX" |-> 64'0
    "RCX" |-> 64'0
    "RDI" |-> 64'0
    "RDX" |-> 64'0
    "RIP" |-> 64'4198400
    "RSI" |-> 64'0
    "RSP" |-> 64'0
    "SF" |-> 1'0
    "SMAP" |-> 1'0
    "SMEP" |-> 1'0
    "SMXE" |-> 1'0
    "SS" |-> 16'0
    "TPL" |-> 4'0
    "TS" |-> 1'0
    "TSD" |-> 1'0
    "UMIP" |-> 1'0
    "VME" |-> 1'0
    "VMXE" |-> 1'0
    "WP" |-> 1'0
    "YMM0" |-> 256'0
    "YMM1" |-> 256'0
    "YMM2" |-> 256'0
    "YMM3" |-> 256'0
    "YMM4" |-> 256'0
    "YMM5" |-> 256'0
    "YMM6" |-> 256'0
    "YMM7" |-> 256'0
    "YMM8" |-> 256'0
    "YMM9" |-> 256'0
    "YMM10" |-> 256'0
    "YMM11" |-> 256'0
    "YMM12" |-> 256'0
    "YMM13" |-> 256'0
    "YMM14" |-> 256'0
    "YMM15" |-> 256'0
    "ZF" |-> 1'1
  </regstate>
  <msrmapping>
    32'3221225600 |-> "IA32_EFER"
  </msrmapping>
  <regstatequeue>
    ListItem ( "AF" |-> 1'0
    "AM" |-> 1'0
    "CD" |-> 1'0
    "CF" |-> 1'0
    "CR2" |-> 32'0
    "CS" |-> 16'0
    "DE" |-> 1'0
    "DF" |-> 1'0
    "DS" |-> 16'0
    "EFLAGS" |-> 32'0
    "EM" |-> 1'0
    "ES" |-> 16'0
    "ET" |-> 1'1
    "FS" |-> 16'0
    "FSGBASE" |-> 1'0
    "GDTR_BASE" |-> 64'0
    "GDTR_LIMIT" |-> 16'0
    "GS" |-> 16'0
    "IA32_EFER" |-> 64'0
    "MCE" |-> 1'0
    "MP" |-> 1'0
    "NE" |-> 1'0
    "NW" |-> 1'0
    "OF" |-> 1'0
    "OSFXSR" |-> 1'0
    "OSXMMEXCPT" |-> 1'0
    "OSXSAVE" |-> 1'0
    "PAE" |-> 1'0
    "PCD" |-> 1'0
    "PCE" |-> 1'0
    "PCIDE" |-> 1'0
    "PE" |-> 1'1
    "PF" |-> 1'1
    "PG" |-> 1'0
    "PGE" |-> 1'0
    "PKE" |-> 1'0
    "PSE" |-> 1'0
    "PVI" |-> 1'0
    "PWT" |-> 1'0
    "PageDirectoryBase" |-> 20'0
    "R8" |-> 64'0
    "R9" |-> 64'0
    "R10" |-> 64'0
    "R11" |-> 64'0
    "R12" |-> 64'0
    "R13" |-> 64'0
    "R14" |-> 64'0
    "R15" |-> 64'0
    "RAX" |-> 64'0
    "RBP" |-> 64'0
    "RBX" |-> 64'0
    "RCX" |-> 64'0
    "RDI" |-> 64'0
    "RDX" |-> 64'0
    "RIP" |-> 64'4198400
    "RSI" |-> 64'0
    "RSP" |-> 64'0
    "SF" |-> 1'0
    "SMAP" |-> 1'0
    "SMEP" |-> 1'0
    "SMXE" |-> 1'0
    "SS" |-> 16'0
    "TPL" |-> 4'0
    "TS" |-> 1'0
    "TSD" |-> 1'0
    "UMIP" |-> 1'0
    "VME" |-> 1'0
    "VMXE" |-> 1'0
    "WP" |-> 1'0
    "YMM0" |-> 256'0
    "YMM1" |-> 256'0
    "YMM2" |-> 256'0
    "YMM3" |-> 256'0
    "YMM4" |-> 256'0
    "YMM5" |-> 256'0
    "YMM6" |-> 256'0
    "YMM7" |-> 256'0
    "YMM8" |-> 256'0
    "YMM9" |-> 256'0
    "YMM10" |-> 256'0
    "YMM11" |-> 256'0
    "YMM12" |-> 256'0
    "YMM13" |-> 256'0
    "YMM14" |-> 256'0
    "YMM15" |-> 256'0
    "ZF" |-> 1'1 )
    ListItem ( .Map )
  </regstatequeue>
  <formatcountint>
    0
  </formatcountint>
  <formatcountfloat>
    0
  </formatcountfloat>
</generatedTop>
